### DISCUSSION TITLE
CLI Commands to add proxy hosts?

### DISCUSSION THREAD

**Discussion by maurxce** [NginxProxyManager/nginx-proxy-manager#3298] [Q&A]
Hey all!
I've been using NPM for some time now and really like it. I'm currently reworking my server structure and wanted to create something like a quick-deploy script for basic functionalities.

Now that has me wondering if there are any commands to create proxy hosts from the CLI? This would make the set up of my deployment way faster

Thanks in advance!

**Comment by maurxce** [comment]
^ TLDR: Commands could benefit automated set ups

**Comment by GregJohnStewart** [comment]
Bump! I pretty much need this for my usecase.. would be fantastic to have!
